types/Transform.js

```diff
@@ -11,6 +11,11 @@
 
 import type {Config, Path} from 'types/Config';
 
+export type TransformedSource = {|
+  content: string,
+  sourceMap: ?Object | string,
```

Comment:
No, typescript returns something different. I was trying not to play favorites with compilers *too* much. I'll change to babel's shape.